app/assets/stylesheets/application/contributors.scss

```diff
@@ -1,9 +1,49 @@
-.hideContent {
-  display: none;
-}
+.contributorModalBody {
+	display: flex;
+	flex-direction: row;
+
+	.contributorModalBodyPhoto {
+    flex: 0 1 30%;
+    @media screen and (max-width: $small) {
+      flex: 0 1 25%;
+      .contributorModalAvatarMedium img.Avatar__image___2cXeU {
+        width: 100px;
+        height: 100px;
+      }
+    }
+  }
+
+	.contributorModalBodyCopy {
+    margin-left: $size-18;
+    
+    .contributorModalHeader {
+      display: flex;
+      align-items: center;
```

Comment:
At smaller breakpoints, would love to stack the avatar, name, and social links on top of one another, so perhaps a `flex-direction: column;` would be appropriate here! Reverting to `flex-direction: row;` for larger breakpoints is totally fine, to keep it looking like how it did before.